backend/app/rest/tag_routes.py

```diff
@@ -0,0 +1,20 @@
+from flask import Blueprint, current_app, jsonify
+from ..middlewares.auth import require_authorization_by_role
+from ..services.implementations.tags_service import TagsService
+
+tags_service = TagsService(current_app.logger)
```

Comment:
nit: For `tag_routes.py` and `tags_service.py` I think both names should have tags be plural